index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <title> home </title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="./css/styles.css"/>
+    </head>
+    <body>
+        <header>
+            <nav>
+                <div class="brand">
+                    <h1> My Logo </h1>
+                </div>
+                <div>
+                    <img src="./images/dashicons_menu.png" alt="dashicon menu icon"/>
+                </div>
+            </nav>
+        </header>
+        <section class="intro">
+            <div class="title">
+                <h1> I’m Mariane Glad to see you! </h1>               
+            </div>
+            <div>
+                <article>
+                    I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </article>
+            </div>
+            <div>
+                <h2 class="connect"> LET’S CONNECT </h2>
+            </div>
+            <section class="icons">
+                <div>
+                    <img src="./images/twitter.svg" alt="twitter logo"/>
+                </div>
+                <div>
+                    <img src="./images/Linkedin_icon.svg" alt="linkedin logo"/>
+                </div>
+                <div>
+                    <img src="./images/medium.svg" alt="medium logo"/>
+                </div>
+                <div>
+                    <img src="./images/github.svg" alt="github logo"/>
+                </div>
+                <div>
+                    <img src="./images/Vector.svg" alt="vector logo"/>
+                </div>
+            </section>
```

Comment:
Use the `<ul>` tag to group the list of icons and `<li>` per each icon not <div> tag. Each icon should be a link: you can make it a link by using the `<a>` tag.